datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingListener.java

```diff
@@ -9,7 +9,7 @@
  * {@link DataCapturingService#start(DataCapturingListener,de.cyface.datacapturing.model.Vehicle)}.
  *
  * @author Klemens Muthmann
- * @version 1.1.0
+ * @version 1.2.0
```

Comment:
Why did the code version change but nothing else?

Reply:
Because I reverted all changes in a following commit by hand.